client/pokemon/pokemon-full.view.html

```diff
@@ -1,3 +1,28 @@
+<md-toolbar class="md-hue-1">
+  <div class="md-toolbar-tools">
+    <div ng-if="pokemon.data.box" style="margin-left:-15px;">
+      <a ng-href="/#/box/{{pokemon.data.box}}">
+        <md-button ng-show="pokemon.data.owner === main.user.name || main.user.isAdmin" class="md-icon-button" aria-label="Back to box">
+          <md-tooltip>Back to box</md-tooltip>
+          <i class="material-icons">arrow_back</i>
+        </md-button>
+      </a>
+    </div>
+    <div class="icon-container">
+      <span>
+        <img ng-if="pokemon.data.isEgg" style="left:-10px" src="/img/icons/egg.png" />
+        <img ng-class="{'pokemon-egg': pokemon.data.isEgg}" ng-src="/img/icons/{{pokemon.iconUrl}}.png" />
+        <h3 style="margin-left:40px;">{{pokemon.parsedNickname}}</h3>
+      </span>
+    </div>
+    <span flex></span> <!-- can remove this when we add menu -->
+    <md-button class="md-icon-button" aria-label="Pokémon settings" ng-show="pokemon.data.owner === main.user.name || main.user.isAdmin">
+      <md-tooltip md-direction="left">{{pokemon.parsedNickname}}'s settings</md-tooltip>
+      <i class="material-icons">settings</i>
+    </md-button>
+  </div>
+</md-toolbar>
```

Comment:
Can you put this stuff in the div with the `ng-show="pokemon.data.id"` thing? At the moment the bar shows if there is a 404 error, and it also loads a blank bar while the server is sending a response.